### PR TITLE
keep scheduled date for users without permission to edit it

### PR DESCRIPTION
**Pull request by herbdool**
Addresses https://github.com/backdrop/backdrop-issues/issues/3731

### CODE REVIEW

File: core/modules/node/node.pages.inc [hunk 1]
```diff
@@ -330,7 +330,7 @@ function node_form($form, &$form_state, Node $node) {
     $form['options']['status'][NODE_SCHEDULED]['#access'] = FALSE;
   }
 
-  if ($node_type->settings['scheduling_enabled'] && user_access('administer nodes')) {
+  if ($node_type->settings['scheduling_enabled']) {
```
Comment (quicksketch) [inline core/modules/node/node.pages.inc:333]: This IF statement is now the same condition as the one a few lines earlier (below `Adjustments to the status radios if scheduling is enabled/disabled.`). We should consolidate these into a single IF/ELSE.

File: core/modules/node/node.pages.inc [hunk 2]
```diff
@@ -349,7 +349,7 @@ function node_form($form, &$form_state, Node $node) {
   }
   // If the user doesn't have access to scheduling, default to published.
   elseif ($form['options']['status']['#default_value'] == NODE_SCHEDULED) {
-    $form['options']['status']['#default_value'] = NODE_PUBLISHED;
+    $form['options']['status']['#default_value'] = $node->nid ? $status : NODE_PUBLISHED;
```
Comment (quicksketch) [inline core/modules/node/node.pages.inc:352]: I'm not sure this line is necessary any more after these changes. The default value should already be accurate when $status is calculated earlier above.

File: core/modules/node/node.pages.inc (hunk 1 above)
Comment (jenlampton) [reply core/modules/node/node.pages.inc:333]: Fixed in https://github.com/backdrop/backdrop/pull/2683

File: core/modules/node/node.pages.inc (hunk 2 above)
Comment (jenlampton) [reply core/modules/node/node.pages.inc:352]: Yep, it seems to work without it.

**Comment by jenlampton** [comment]
Replaced by https://github.com/backdrop/backdrop/pull/2683

File: core/modules/node/node.pages.inc (hunk 2 above)
Comment (herbdool) [reply core/modules/node/node.pages.inc:352]: @jenlampton in your new PR I think you left it in. I tested it with it in but not with it gone.

Comment (jenlampton) [reply core/modules/node/node.pages.inc:352]: you're right - force pushed to PR.

**Pull request closed without merging**